packages/components/src/lib/hooks/useSquiggle.ts

```diff
@@ -9,13 +10,31 @@ import { useEffect, useMemo } from "react";
 import { JsImports, jsImportsToSquiggleCode } from "../jsImports";
 import * as uuid from "uuid";
 
-type SquiggleArgs = {
+export type SquiggleArgs = {
   code: string;
   executionId?: number;
   jsImports?: JsImports;
+  onChange?: (
+    expr: result<SqValue, SqError> | undefined,
```

Comment:
Oh I don't like this, I need to change this back. Sorry! This was a different thought (using onChange to allow SquiggleEditor to import SquiggleChart again). However I made SquiggleEditor and SquiggleChart small enough that the duplication isn't that much of an issue.